### DISCUSSION TITLE
TIMER ESP32 too many arguments

### DISCUSSION THREAD

**Discussion by aruffato** [espressif/arduino-esp32#9710] [Q&A]
Good moorning. I'm ADRIAN from Argentina, SPANISH spoken.
What happens with last UPDATE?
Now I have an error with ->timerBegin(1, 80, true) ->error: too many arguments to function 'hw_timer_t* timerBegin(uint32_t)'
Yesterday, I have no problem with the compiler

**Comment by davidcyr2000** [comment]
See https://github.com/adafruit/RadioHead/issues/79
